server/src/main/scala/cromwell/CromwellEntryPoint.scala

```diff
@@ -44,6 +44,10 @@ object CromwellEntryPoint extends GracefulStopSupport {
 
   val gracefulShutdown = config.as[Boolean]("system.graceful-server-shutdown")
 
+  // 3 minute DNS TTL down from JVM default of infinite [BA-6454]
+  val dnsCacheTtl = config.getOrElse("system.dns-cache-ttl", 3 minutes)
```

Comment:
TOL the "or else" aspect of this becomes redundant with the addition in `reference.conf`, but it's not really hurting anything either